src/main/java/net/imagej/ops/commands/filter/FrangiVesselness.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * #%L
+ * ImageJ software for multidimensional image processing and analysis.
+ * %%
+ * Copyright (C) 2014 - 2017 Board of Regents of the University of
+ * Wisconsin-Madison, University of Konstanz and Brian Northan.
+ * %%
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package net.imagej.ops.commands.filter;
+
+import java.util.Arrays;
+
+import net.imagej.ops.OpService;
+import net.imglib2.Dimensions;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.img.Img;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.type.numeric.real.FloatType;
+import net.imglib2.view.IntervalView;
+import net.imglib2.view.Views;
+
+import org.scijava.ItemIO;
+import org.scijava.command.Command;
+import org.scijava.log.LogService;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+/**
+ * This command applied the Frangi Vesselness filter operation onto an image,
+ * allowing users to specify:
+ * 		1) The scales desired for the filter application
+ * 		2) The physical spacing between image data points
+ * 		3) Whether or not the pre-Gaussian filter should be performed.
+ * 
+ * @author Gabe Selzer
+ */
+@Plugin(type = Command.class, menuPath = "Process>Filters>Frangi Vesselness")
+public class FrangiVesselness<T extends RealType<T>> implements Command {
+
+	@Parameter
+	private LogService log;
+
+	@Parameter
+	private OpService opService;
+
+	@Parameter
+	private Img<T> input;
+
+	@Parameter(label = "Gauss before:")
+	private boolean doGauss;
+
+	@Parameter(label = "Spacing",
+		description = "Physical distance between the observed data points")
```

Comment:
Doesn't "Physical distance" imply something like 3 microns? From the implementation, I'd say that is more of a "relative distance": `1,1` implies that we have the same spacing in x and y, whereas `1,1,2` implies that the resolution in z is half of the resolution in x and y.

Reply:
Yes, that is a better way of expressing that.